feat: blog post on query language (#26)
someengineering/resoto.com: blog/2022-02-04-resoto-query-language-101/index.md

<Image img={require('./img/banner.png')} alt="Left: Sheep Spinning Up Cloud Resources; Right: Confused Sheep with Abacus" />

> Retrieving information about resources you have deployed in your [AWS](https://aws.amazon.com) infrastructure means tediously navigating the [AWS](https://aws.amazon.com) Management Console or using the AWS Command Line Interface. That approach works well in a single account setup. However, the recommended best practice for [AWS](https://aws.amazon.com) is to set up a multi-account environment - and as the number of accounts grows, navigating your infrastructure and finding resources via the Console or the CLI becomes hard to impossible.

Author: ```suggestion
Retrieving information about resources you have deployed in your [AWS](https://aws.amazon.com) infrastructure means tediously navigating the [AWS Management Console](https://aws.amazon.com/console) or using the [AWS Command Line Interface](https://aws.amazon.com/cli). That approach works well in a single account setup. However, the recommended best practice for [AWS](https://aws.amazon.com) is to set up a multi-account environment - and as the number of accounts grows, navigating your infrastructure and finding resources via the [Console](https://aws.amazon.com/console) or the [CLI](https://aws.amazon.com/cli/) becomes hard to impossible.
```

> Retrieving information about resources you have deployed in your [AWS](https://aws.amazon.com) infrastructure means tediously navigating the [AWS Management Console](https://aws.amazon.com/console) or using the [AWS Command Line Interface](https://aws.amazon.com/cli). That approach works well in a single account setup. However, the recommended best practice for [AWS](https://aws.amazon.com) is to set up a multi-account environment - and as the number of accounts grows, navigating your infrastructure and finding resources via the [Console](https://aws.amazon.com/console) or the [CLI](https://aws.amazon.com/cli/) becomes hard to impossible.

Author: ```suggestion
Retrieving information about resources you have deployed in your [AWS](https://aws.amazon.com) infrastructure means tediously navigating the [AWS Management Console](https://aws.amazon.com/console) or using the [AWS Command Line Interface](https://aws.amazon.com/cli). This approach works well in a single account setup, but best practice is to set up a multi-account environment. As the number of accounts grows, navigating your infrastructure and finding resources via the [Console](https://aws.amazon.com/console) or the [CLI](https://aws.amazon.com/cli/) becomes hard to impossible.
```
